models/model1_part1.ipynb

```diff
@@ -0,0 +1,567 @@
+{
```

Comment:
<p>This is a well written block.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PanwarJayant/spot/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>